e2e/testcafe-devextreme/tests/dataGrid/editing.ts

```diff
@@ -2701,3 +2701,27 @@ test('Focus behavior should be correct when editing cells', async (t) => {
     mode: 'batch',
   },
 }));
+
+test('DataGrid - Revert button flickers in `cell` editing mode (T1247323)', async (t) => {
+  const dataGrid = new DataGrid('#container');
+  const revertButton = Selector('.dx-revert-button');
```

Comment:
Please use [the model method](https://github.com/DevExpress/DevExtreme/blob/3d11172b99a5750ce41b12367c39d4d87a5b2ebb/packages/testcafe-models/dataGrid/index.ts#L373)